lang/php/Portfile

```diff
@@ -1206,6 +1215,22 @@ subport ${php}-intl {
 
     depends_lib-append      path:lib/pkgconfig/icu-uc.pc:icu
 
+    if {[vercmp ${branch} 5.3] >= 0 && [vercmp ${branch} 7.0] < 0} {
+        patchfiles-append   patch-${php}-intl-icu-68.diff
+    }
+
+    if {[vercmp ${branch} 5.5] >= 0 && [vercmp ${branch} 7.4] < 0} {
+        patchfiles-append   patch-php7-intl-icu-71.diff
+    }
+
+    if {[vercmp ${branch} 7.0] >= 0 && [vercmp ${branch} 7.3] < 0} {
+        patchfiles-append   patch-php7-intl-icu-56.diff
+    }
+
+    if {[vercmp ${branch} 7.0] >= 0 && [vercmp ${branch} 8.0] < 0} {
+        patchfiles-append   patch-php7-intl-icu-68.diff
+    }
```

Comment:
I'm not thrilled by the introduction of the "php7-" prefix in these patchfile names. It's not accurate, and all other patchfiles in this port are for individual php major versions, and I'd like to retain that here as well, even if that means just duplicating the patchfiles under different names.

Reply:
got it